ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java

```diff
@@ -360,4 +369,21 @@ public static void addMethods(
       methods.put(rpcMethod.getName(), rpcMethod);
     }
   }
+
+  private PrivateMarkerTransactionFactory createPrivateMarkerTransactionFactory(
+      final PrivacyParameters privacyParameters,
+      final BlockchainQueries blockchain,
```

Comment:
Variable name is bit misleading, blockchainQueries?

Reply:
done.